step.yml

```diff
@@ -1,9 +1,24 @@
-title: Authenticate host with netrc
-summary: Adds your authentication config to the .netrc file
+title: "Authenticate host with netrc"
+summary: |-
+  Adds your authentication configuration to the `.netrc` file.
+
 description: |-
-  Adds your authentication config to the .netrc file.
-  
-  If you already have a .netrc file, the step will create a backup of the original, and appends the configs to the current one.
+  [This Step](https://github.com/bitrise-steplib/steps-authenticate-host-with-netrc) adds the authentication configuration (host name, login name and password string) to the `.netrc` file .
+  The Step lets you store your remote credentials on the build VM once so that later steps can use the credentials for authentication instead of requiring manual input. Examples include HTTPS git clone URLs with OAuth token-based authentication (instead of authenticating with SSH key).
```

Comment:
The Step lets you store your remote credentials on the build VM once, so that later Steps can use the credentials for authentication instead of requiring manual input. Possible examples include HTTPS git clone URLs with OAuth token-based authentication (instead of authenticating with SSH key).